OpenSim/Simulation/Model/ControllerSet.h

```diff
@@ -25,6 +25,7 @@
 
 
 // INCLUDES
+#include "OpenSim/Common/TimeSeriesTable.h"
```

Comment:
I don't think so. The one with quotes searches more directories than one with angle brackets.